cleanup.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
Please add this single use commands to your run-cleanup make target so you can use them inside the container and I can use my former make targets outside.

Reply:
See other comment for the makefile. Regardless of that I would like to have this bash file since I use that all the time when I'm in the docker container ;)